packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -37,10 +37,16 @@ class CreateCommand extends FlutterCommand {
       help: 'Also add a flutter_driver dependency and generate a sample \'flutter drive\' test.'
     );
     argParser.addFlag(
-      'plugin',
+      'package',
       negatable: true,
       defaultsTo: false,
-      help: 'Generate a Flutter plugin project.'
+      help: 'Generate a Flutter package project.'
```

Comment:
We can use the following help strings:

Package: "Generate a shareable Flutter project containing modular Dart code."

Plugin: "Generate a shareable Flutter project containing an API in Dart code with a platform-specific implementation for Android, for iOS code, or for both."